packager/mpd/test/xml_compare.cc

```diff
@@ -80,10 +78,17 @@ bool CompareContents(xmlNodePtr node1, xmlNodePtr node2) {
       reinterpret_cast<const char*>(node1_content_ptr.get());
   std::string node2_content =
       reinterpret_cast<const char*>(node2_content_ptr.get());
-  base::ReplaceChars(node1_content, "\n", "", &node1_content);
-  base::TrimString(node1_content, " ", &node1_content);
-  base::ReplaceChars(node2_content, "\n", "", &node2_content);
-  base::TrimString(node2_content, " ", &node2_content);
+
+  node1_content.erase(
```

Comment:
this doesn't feel like the best way to do the equivalent of `base::ReplaceChars` but it works so I started with this

Reply:
LGTM.  It's a test helper, so it's fine.